var/spack/repos/builtin/packages/ldak/package.py

```diff
@@ -12,31 +12,68 @@ class Ldak(Package):
     homepage = "https://dougspeed.com/ldak/"
     url = "https://dougspeed.com/wp-content/uploads/source.zip"
 
-    version("5.1", sha256="ae3eb8c2ef31af210e138336fd6edcd0e3a26ea9bae89fd6c0c6ea33e3a1517e")
+    maintainers("snehring")
 
-    variant("mkl", default=False, description="Use MKL")
+    version("5.2", sha256="ba3de4eb4f2d664b3c2a54bef2eb66d1a498ac423179e97a5795d010161b1805")
+    version(
+        "5.1",
+        sha256="ae3eb8c2ef31af210e138336fd6edcd0e3a26ea9bae89fd6c0c6ea33e3a1517e",
+        deprecated=True,
+    )
```

Comment:
@snehring are we deprecating v5.1 due to a CVE? Or any particular reason? Else we should keep it in as a regular version.

Reply:
Only one version of the source seems to be distributed at a time, so the URL is currently serving the latest version.

Reply:
Oh man, I kinda hate that. Since there's no way to get the older source version, let's remove the version all together. In the future we should reach out to the upstream developer and see if they can keep older versions around with unique paths.

Reply:
I think we still have the old version in the spack source mirror, so ostensibly it is still installable from there. I'm willing to go either way though, let me know if that changes your mind.

Reply:
Good question, can you quickly check if Spack finds the cached version in the source mirror. I'm not sure if sharing a URL might break that system.

Reply:
It does pull from the source cache with a freshly cloned from develop version of spack.